packages/mwp-app-render/src/components/SEOHead.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+
+import {
+	DEFAULT_IMAGE_URL,
+	DEFAULT_TITLE,
+	generateMetaData,
+	generateCanonicalUrlMetaTags,
+	generateMetaTags,
+} from '../util/seoHelper';
+
+/**
+ * Component for rendering SEO content in the document's head
+ * Note: props passed to SEOHead should *not* be html-escaped because this is handled by Helmet
+ * @see https://github.com/nfl/react-helmet
+ * @param {String} baseUrl Base url of the page (protocol + hostname).
+ * @param {String} imageUrl An image url that will be applied to the link tag, og:image meta tag, etc.
+ * @param {Array} ldJson Array containing json-ld objects. These are usually generated by helper functions in src/util/seoHelper.js.
+ * @param {String} localCode The current localeCode.
+ * @param {String} ogDescription A special description for the Open Graph og:description meta tag. If this isn't provided pageDescription is used.
+ * @param {String} ogTitle A special title for the Open Graph og:title meta tag. If this isn't provided pageTitle is used.
+ * @param {String} pageDescription The page description, for example -- Upcoming Events for NY Tech in New York City
+ * @param {String} pageKeywords A comma-separated string of keywords, for example -- 'upcoming events,ny-tech,ny,technology,group,club,event,community,local,networking,meet,sharing'
+ * @param {Array} pageMeta Optional meta tag data, for example -- [{name: 'yandex', value: 12345},{name: 'bing', value: 12345}]
+ * @param {String} pageTitle Content for the page's title tag
+ * @param {Boolean} robots Instructs search engines whether or not we'd like them to crawl the pages and its links
+ * @param {String} route The current route
+ * @module SEOHead
+ */
+export const SEOHeadComponent = ({
+	baseUrl,
+	imageUrl,
+	ldJson,
+	localeCode,
+	ogDescription,
+	ogTitle,
+	pageDescription,
+	pageKeywords,
+	pageMeta,
+	pageTitle,
+	robots,
+	route,
+}) => {
+
+	const metaData = generateMetaData({
+		appPath: `meetup:/${route}`,
+		baseUrl,
+		description: pageDescription,
```

Comment:
I'm not sure renaming `pageDescription` to `description` is beneficial. It might be nice to keep the same name  throughout and we can simplify the code to `{ appPath, baseUrl, pageDescription }`, etc.?

Reply:
yes